### PR TITLE
Translate lzl

### PR DESCRIPTION
**Pull request by jxnclzl**

### CODE REVIEW

File: i18n/en/advance-usage/1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (hobo0cn) [inline i18n/en/advance-usage/1.txt:1]: What this for？

**Review by hobo0cn — COMMENTED** [review]
Where's other translate?

File: i18n/en/advance-usage/1.txt (hunk 1 above)
Comment (jxnclzl) [reply i18n/en/advance-usage/1.txt]: never mind, has been deleted.

**Comment by hobo0cn** [comment]
Please re-create this Pull Request from your fork to origin/develop branch. @jxnclzl

**Comment by hobo0cn** [comment]
We'll review this PR #9

**Pull request closed without merging**